docs/commands.md

```diff
@@ -0,0 +1,65 @@
+---
+id: commands
+title: Command Line Interface Reference
+sidebar_label: CLI Reference
+---
+
+This page is a quick reference for all of the command that are used to manage the application.
+
+## Setup Command
+
+The setup command loads the website configuration and creates the database. It can only be run once (unless the database
```

Comment:
```suggestion
The `setup` command loads the website configuration and creates the database. It can only be run once (unless the database
```
Suggest we 'mark' the commands, either by backticks or bold? I'll suggest this for other commands; your choice to ignore if you don't think its a good idea.